docs/spark-queries.md

```diff
@@ -346,6 +346,9 @@ SELECT * FROM prod.db.table.partitions;
 Note:
 For unpartitioned tables, the partitions table will contain only the record_count and file_count columns.
 
+Note2:
```

Comment:
Can you make the two notes a bullet list.  See other notes like Manifests / All Manifests

Reply:
Sure, done.